issue-29 - Add new annotation to ignore generation

tfdsimoes opened the pull request.

Commit:
issue-29 - Add new annotation to ignore generation

Commit:
issue-29 - Correction PR for Codacy

jemacineiras committed:
Merge branch 'main' into feature/29-include-a-new-annotation-to-ignore-fields-in-dsl-generation

Commit:
Merge branch 'main' into feature/29-include-a-new-annotation-to-ignore-fields-in-dsl-generation

Commit:
issue-29 - Update solution for List, Map and Set. Plus refactor of previous solution

Commit:
Merge remote-tracking branch 'origin/feature/29-include-a-new-annotation-to-ignore-fields-in-dsl-generation' into feature/29-include-a-new-annotation-to-ignore-fields-in-dsl-generation

Commit:
issue 29 - Correction of @DslExclude

Commit:
Merge branch 'main' into feature/29-include-a-new-annotation-to-ignore-fields-in-dsl-generation

# Conflicts:
#	pom.xml

Commit:
issue - 29 - Add new annotation into Readme

tfdsimoes commented:
The errors in codacy is because I do not have the code style you use., so the imports are in the wrong order.

apenlor commented:
> The errors in codacy is because I do not have the code style you use., so the imports are in the wrong order.

PR created for including not only checkstyle but also the Idea style we're using in the project
https://github.com/sngular/pact-annotation-processor/pull/38

Commit:
Merge branch 'main' into feature/29-include-a-new-annotation-to-ignore-fields-in-dsl-generation

Commit:
issue - 29 - Correction several style errors

tfdsimoes commented:
The error in codacy is related to README, a new line with more than 180 of lenght

apenlor committed:
Merge branch 'main' into feature/29-include-a-new-annotation-to-ignore-fields-in-dsl-generation

apenlor committed:
DslExclude fix & integration with buildExpectedInstace feature

- Fixes template to avoid including excluded fields in dsl with empty values
- Integrates DslExclude with the latest buildExpectedInstace method

apenlor committed:
relocate checkstyle.xml
